js-jest/test/gilded_rose.test.js

```diff
@@ -78,22 +69,24 @@ describe('general rules for items in store', () => {
     expect(updateItemQuality).toThrowError('Item Quality parameter should be a whole number between 0 and 50');
   })
 
-  // it('should lower its Sellin value when the updateQuality function is run', () => {
-  //   const initialItem = new Item('foo', 5, 10)
-  //   const gildedRose = new Shop([{...initialItem}]);
-  //   const items = gildedRose.updateQuality();
-  
-  //   expect(items[0].sellIn).toBeLessThan(initialItem);
-
-  // })
-
-  // it('should lower its Quality value when the updateQuality function is run', () => {
-
-  // })
-
-  // it('quality should degrade twice as fast after sell by date', () => {
+  it('Sellin value should reduce when the updateQuality function is run', () => {
```

Comment:
Your unit tests are naturally structured in the arrange-act-assert pattern (http://wiki.c2.com/?ArrangeActAssert) 👍 

Do you have any thoughts on whether adding the whitespace between the phases helps readability?